angular/projects/admin-nrpti/src/app/records/restorative-justices/restorative-justice-add-edit/restorative-justice-add-edit.component.html

```diff
@@ -87,8 +87,16 @@ <h4>Basic Information</h4>
     </section>
 
     <section>
-      <h4>Legislation</h4>
+      <h4>Contravention</h4>
       <app-legislation-add-edit [formGroup]="myForm"></app-legislation-add-edit>
+      <div class="label-pair">
+        <label for="offence">Offence</label>
+        <input name="offence" id="offence" formControlName="offence" class="form-control" />
+      </div>
+      <div class="label-pair">
+        <label for="penalty">Penalty</label>
+        <input name="penalty" id="penalty" formControlName="penalty" class="form-control" />
```

Comment:
Depending on this comment: https://github.com/bcgov/NRPTI/pull/262#discussion_r400516207 . This probably should be limited to only numbers.

Reply:
This will be resolved in a separate PR.